src/main/lrsql/init/localization.clj

```diff
@@ -0,0 +1,14 @@
+(ns lrsql.init.localization
+  "Utilities for white labeling/localization overrides"
+  (:require [cheshire.core :as json]
+            [clojure.java.io :as io]))
+
+(def lang-path
+  "lrsql/localization/language.json")
+
+(def custom-language-map
```

Comment:
As discussed on slack, put this in a memoized function to call when serving the `GET /env` request. That should make this safe for AOT and still allow override at runtime.